packages/babel-plugin/src/babel-plugin.ts

```diff
@@ -223,6 +288,22 @@ export default declare<State>((api) => {
         path: NodePath<t.TaggedTemplateExpression> | NodePath<t.CallExpression>,
         state: State
       ) {
+        if (isJsx(path, state)) {
+          throw buildCodeFrameError(
+            `Found a \`jsx\` function call in the Babel output where one should not have been generated. Was Compiled was not set up correctly?
+
+Reasons this might happen:
+* Importing \`jsx\` from a library other than Compiled CSS-in-JS - please only import from \`@compiled/react\`.
+* Both \`runtime: classic\` and \`pragma: 'jsx'\` were specified in your Babel configuration - please use the /** @jsx jsx */ syntax instead, or switch to \`runtime: automatic\`.
+
+See https://compiledcssinjs.com/docs/installation for how to set up the Compiled Babel plugin.`,
```

Comment:
nit: Should we mention anything around Webpack / Parcel?

Reply:
hmm probs not necessary?

`@babel/plugin-transform-react-jsx` is the plugin where one would specify `runtime: classic` and `pragma: 'jsx'` (and potentially run into this error), and

* I think with parcel you would need a `babel.config.json` to specify the above config options (not 100% sure on this, best to fact-check haha)
* with webpack you can include the config options within your `webpack.config.js` (e.g. [this person's setup](https://stackoverflow.com/q/52565118)) but it's probably clear enough what "Babel configuration" would refer to here

Reply:
I guess the point I'm making is most people wouldn't use the Babel plugin directly right, but through a bundler plugin?

Reply:
updated message for clarity 👍 let me know what you think